.github/dependabot.yml

```diff
@@ -4,5 +4,5 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
-    ignore:
-      - dependency-type: "development"
+    allow:
+      - dependency-type: "production"
```

Comment:
The previous syntax:
```yml
ignore:
      - dependency-type: "development"
```

…caused a [GitHub action error](https://github.com/kienstra/machine-learning/runs/8705619252):
The property '#/updates/0/ignore/0' contains additional properties ["dependency-type"] outside of the schema when none are allowed
The property '#/updates/0/ignore/0' did not contain a required property of 'dependency-name'